CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Grafana Mimir
 
+* [CHANGE] Query-frontend: truncate queries based on the configured creation grace period (`--validation.create-grace-period`) to avoid querying too far into the future. #3172
```

Comment:
I would say this is an ENHANCEMENT, not a CHANGE, as per https://github.com/grafana/mimir/blob/main/docs/internal/contributing/README.md#changelog

Reply:
Updated.